src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14InstanceofWithPatternMatching.java

```diff
@@ -1,9 +1,13 @@
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.grammar.java14;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
 import java.util.Locale;
 
+import org.w3c.dom.ls.LSOutput;
```

Comment:
This import is not used. What is it for?

Reply:
`java.util.List` is also not used.

Reply:
Fixed.